src/formatter.ts

```diff
@@ -5,6 +5,7 @@ import type { AttributeValue, Function, Node, Value, Variable } from './types';
 type Options = {
   allowIndentation?: boolean;
   maxTagOpeningWidth?: number;
+  incrementListNumbers?: boolean;
```

Comment:
Lurk: I wonder if this parameter should be a string union type instead? Something like: `orderedListMode: "increment"` or something?

Reply:
Cc @bibek-stripe @matv-stripe

Reply:
I like that too!

Reply:
@mfix-stripe do you have other modes in mind?

Reply:
Not at the moment, I just always tend to lean away from booleans for future-proofing. I could see someone wanting something else in the future.